waltid-wallet-api/k8s/deployment-dev.yaml

```diff
@@ -262,6 +266,9 @@ spec:
         - name: wallet-oci-secret
```

Comment:
may I suggest to add a config-map for wallet-oci-secret (perhaps have it renamed to wallet-oci-config) here, as it doesn't contain any secrets anymore?